files/motd.txt

```diff
@@ -0,0 +1,23 @@
+ _____     _  ___       _     _                
+|  ___|   | |/ / |_   _| |__ | |__   ___ _ __  
+| |_ _____| ' /| | | | | '_ \| '_ \ / _ \ '_ \ 
+|  _|_____| . \| | |_| | |_) | |_) |  __/ | | |
+|_|       |_|\_\_|\__,_|_.__/|_.__/ \___|_| |_|
+                                               
+   _  _   _____ ____  ___ _____ _____ ___ _____ 
+ _| || |_|  ___|  _ \|_ _|_   _|  ___|_ _|_   _|
+|_  ..  _| |_  | |_) || |  | | | |_   | |  | |  
+|_      _|  _| |  _ < | |  | | |  _|  | |  | |  
+  |_||_| |_|   |_| \_\___| |_| |_|   |___| |_| 
+                 
+F-klubben's Server
+OS:             NixOS-23.11
+
+Documentation:  file:///run/current-system/sw/share/doc/nixos/index.html
```

Comment:
Viewing this seems impractical when accessing over SSH, might be better to point to the version on https://nixos.org instead:
```suggestion
Documentation:  https://nixos.org/manual/nixos/stable
```